test/Modules/UngradedModules.jl

```diff
@@ -171,16 +171,20 @@ end
 	B = R[x^2; x*y; y^2; z^4]
 	M = SubquoModule(A, B)
 	free_res = free_resolution(M, length=1)
-    @test is_complete(free_res) == false
-    free_res[2]
-    @test length(free_res.C.maps) == 4
+  @test is_complete(free_res) == false
+  free_res[2]
+  @test length(free_res.C.maps) == 4
```

Comment:
This part seems to be indented weirdly (when viewing it on github). Maybe there is a mixup of tabs and spaces here?

Reply:
There were some 4 space indentations in this file which I removed, everything looks good in my local files.

Reply:
Let's just leave this as is and let https://github.com/oscar-system/Oscar.jl/pull/2785 handle the tab issue.